tests/test-certificates-beacon.sh

```diff
@@ -101,6 +101,24 @@ apply_new_beacon_conf() {
     run_certificates_beacon_state "${pillar[*]}"
 }
 
+reset_beacon_conf() {
+    local salt_container
+
+    salt_container=$(get_salt_container)
+
+    readarray -t minions < <(get_salt_minion_ids)
+
+    for minion in "${minions[@]}"; do
```

Comment:
I'm curious, why do we need a loop here? Can't we just `salt-run metalk8s_saltutil.wait_minions tgt='*' && salt '*' beacons.disable`?